app/views/shared/_show_breadcrumbs.html.erb

```diff
@@ -1,19 +1,20 @@
 <% parents = Arclight::Parents.from_solr_document(document).as_parents %>
 <nav aria-label="breadcrumb">
   <ol class="breadcrumb">
-    <li class="breadcrumb-item breadcrumb-item-1">
-      <%= link_to t('arclight.routes.home'), root_path %>
-    </li>
-    <li class="breadcrumb-item breadcrumb-item-2 al-bc-item">
-      <%= link_to t('arclight.routes.collections'), arclight_engine.collections_path %>
+    <li class="breadcrumb-item al-bc-item breadcrumb-item-1">
+      <% if document.repository_config.present? %>
+        <%= link_to(document.repository_config.name, arclight_engine.repository_path(document.repository_config.slug)) %>
+      <% else %>
+          <span>Repositories</span>
```

Comment:
This string should be internationalized (and singular per @ggeisler's comment).

Reply:
Pushed up these changes and waiting on CI.